common/OpTestIPMI.py

```diff
@@ -554,3 +555,74 @@ def ipmi_get_occ_status(self):
             raise OpTestError(l_msg)
 
         return l_result
+
+    ##
+    # @brief This function will deactivates ipmi sol console
+    #
+    # @return l_con it is a console object or raise OpTestError
+    #
+    def ipmi_sol_deactivate(self):
+        print "running:%s sol deactivate" % self.cv_baseIpmiCmd
+        try:
+            l_con = pexpect.spawn('%s sol deactivate' % self.cv_baseIpmiCmd)
+        except:
+            l_msg = "IPMI: sol deactivate failed"
+            raise OpTestError(l_msg)
+        return l_con
+
+    ##
+    # @brief This function will activates ipmi sol console
+    #
+    # @return l_con it is a console object or raise OpTestError
+    #
+    def ipmi_sol_activate(self):
+        print  "running:%s sol activate" % self.cv_baseIpmiCmd
+        try:
+            l_con = pexpect.spawn('%s sol activate' % self.cv_baseIpmiCmd)
+        except:
+            l_msg = "IPMI: sol activate failed"
+            raise OpTestError(l_msg)
+        return l_con
+
+    ##
+    # @brief This function waits for getting ipmi sol console activated.
+    #
+    # @return l_con it is a console object or raise OpTestError in case of not connecting up to 10mins.
+    #
+    def ipmi_get_console(self):
+        self.ipmi_sol_deactivate()
+        # Waiting for a small time interval as latter versions of ipmi takes a bit of time to deactivate.
+        time.sleep(10)
```

Comment:
its better to use the constants instead of hard coded values. 
Makes it easier to change, if required in future